mail_alias_creator/ldap.py

```diff
@@ -138,8 +138,12 @@ def get_users_in_group(self, group: str) -> List[str]:
                                attributes=[self.group_membership_field]):
                     logger.debug("Found these entries: {}".format(str(conn.entries)))
                     for entry in conn.entries:
-                        for member in entry[self.group_membership_field].value:
-                            results.append(member)
+                        value = entry[self.group_membership_field].value
+                        if type(value) is list:
+                            for member in value:
+                                results.append(member)
+                        elif value is not None: # value is none if no one is in group
```

Comment:
```suggestion
                        elif value is not None: # value is None if the group does not exist or is empty
```

Reply:
```suggestion
                        elif value is not None: # value is None if group is empty
```
If the  group does not exist the `entries` list is empty.